youtube.py

```diff
@@ -50,9 +50,13 @@ def query(self, search_query):
           elif result["id"]["kind"] == "youtube#playlist":
             playlists.append("{} ({})".format (result["snippet"]["title"],result["id"]["playlistId"]))
 
-        print("Videos:\n", "\n".join(videos), "\n")
-        print("Channels:\n", "\n".join(channels), "\n")
-        print("Playlists:\n", "\n".join(playlists), "\n")
+        #print("Videos:\n", "\n".join(videos), "\n")
+        #print("Channels:\n", "\n".join(channels), "\n")
+        #print("Playlists:\n", "\n".join(playlists), "\n")
+
+        results = {"videos": videos, "channels": channels, "playlists": playlists}
```

Comment:
You never wrote a `send_playlist` function, no need to append the `playlists` list to our results if we don't have it (leaving up to you whether you want to implement the `send_playlist` function or to strip playlists from the code entirely)